packages/component-base/src/style-props.js

```diff
@@ -44,11 +44,11 @@ addGlobalThemeStyles(
 
         /* Focus outline */
         --vaadin-focus-ring-width: 2px;
-        --vaadin-focus-ring-color: var(--_vaadin-color);
+        --vaadin-focus-ring-color: var(--_vaadin-color-strong);
```

Comment:
This change seems to break visual tests for unrelated components, it's better to revert it.

Reply:
Let’s do it in a separate PR that updates only that.

Reply:
Yes, I'll create a PR for main branch tomorrow and update all affected screenshots.